osbs/api.py

```diff
@@ -287,6 +288,36 @@ def _create_scratch_build(self, build_request):
         build_json['metadata']['name'] = build_config_name
         return BuildResponse(self.os.create_build(build_json).json())
 
+    def _get_image_stream_info_for_build_request(self, build_request):
+        """Return ImageStream, ImageStreamTag, and ImageStreamTag name
+        for base_image of build_request
+
+        If build_reuqest is not auto instantiated, objects are not fetched
+        and None, None, None is returned.
+        """
+        image_stream = None
+        existing_ist = None
+        image_stream_tag_name = None
+
+        if build_request.is_auto_instantiated():
```

Comment:
Is `is_auto_instantiated()` meant to be accurate at this point? We seem to be doing this calculation across two places. Should it be named `has_ist_trigger()` instead?